hack/update-generated-completions.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STI_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
@rhcarvalho that is 2 :-) I can do a follow up fixing all of them, but that was not the reason I started this PR :-)